menas/ui/service/MappingTableService.js

```diff
@@ -170,8 +170,11 @@ var MappingTableService = new function () {
       if (xhr.status === 400) {
         let err = "Disabling mapping table failed. Clear the following dependencies first:\n";
         let oData = JSON.parse(xhr.responseText);
-        for(let ind in oData) {
-          err += "\t - " + oData[ind].name + " (v. " + oData[ind].version + ")";
+
+        err += "Datasets:\n";
+        let datasets = oData["datasets"];
+        for(let ind in datasets) {
+          err += "- " + datasets[ind].name + " (v. " + datasets[ind].version + ")\n";
```

Comment:
If I understand this correctly, I should not be able to delete a mapping table if there is a dataset that is using it. 

I was able to delete a mapping table while the dataset was using it.

Reply:
Thanks for catching that! I'll prioritize the REST API tests to reduce the odds of things like this happening.

Reply:
Let's log this as a separate issue, it's a back-end problem that I think is addressed in #132 and we can revisit it after.